packages/config/config/devices/0x010f/fgd212_3.5.json

```
@@ -541,62 +491,61 @@
 			"description": "Time required to warm up the filament of halogen bulb.",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 2,
 			"defaultValue": 1,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "no soft-start",
+					"label": "No soft-start",
 					"value": 0
 				},
 				{
-					"label": "short soft-start (0,1s)",
+					"label": "Short soft-start (0,1s)",
 					"value": 1
 				},
 				{
-					"label": "long soft-start (0,5s)",
+					"label": "Long soft-start (0,5s)",
 					"value": 2
 				}
 			]
 		},
 		"35": {
 			"label": "Auto-calibration after power on",
-			"description": "Determines trigger of auto-calibration procedure",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 4,
 			"defaultValue": 1,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "No auto-calibration of the load after power on",
+					"label": "No auto-calibration of after power on",
```

I think we have a good opportunity to shorten the labels here. If the parameter label is just called "Auto-calibration", this could be called
```suggestion
					"label": "Never",
```

---

Nice suggestion, I went with "Perform Auto-calibration" instead of just "Auto-calibration". Ok?